client/src/components/TechVote.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import handleFetchServerData from "../helper/handleFetchServerData";
+import handlePostVoteData from "../helper/handlePostVoteData";
+import DisplayForVote from "./DisplayForVote";
+import "../stylesheets/global.scss";
+import "../stylesheets/global.css";
+import "../stylesheets/TechVote.scss";
+
+class TechVote extends Component {
+  state = {
+    tech_list: [],
+    vote_list: [],
+    flash: "",
+  };
+  componentDidMount() {
+    handleFetchServerData()
+      .then((results) =>
+        this.setState({ tech_list: [...this.state.tech_list, ...results] })
+      )
+      .catch((err) => console.error(err));
+  }
+  storeVote = (technology, vote_type) => {
+    this.setState({
+      vote_list: [
+        ...this.state.vote_list.filter(
+          (vote) => vote.tech_id !== technology.id
+        ),
+        {
+          tech_id: technology.id,
+          vote_type: vote_type,
+        },
+      ],
+    });
+  };
+  handleVoteSubmit = (e) => {
```

Comment:
Very tiny tip, if you add new-lines between different functions, it makes it a bit easier to read :)